<launch>
  <!-- visualization -->
  <node name="stereo_davis_renderer" pkg="dvs_renderer" type="stereo_davis_renderer">
  <!-- output = "screen" -->
    <param name="display_method" value="grayscale" />
    <param name="rectification_method" value="nearest" />
    <param name="integration_length" value="2000" />
    <param name="is_sync" value="true" />
    
    <remap from="dvs_rendering_left" to="davis_left/dvs_rendering" />
    <remap from="dvs_rendering_right" to="davis_right/dvs_rendering" />

    <remap from="dvs_rectified_left" to="davis_left/dvs_rectified" />
    <remap from="dvs_rectified_right" to="davis_right/dvs_rectified" />
    <remap from="dvs_rectified_stereo" to="davis_stereo/dvs_rectified" />
    
    <remap from="image_left" to="/davis_left/image_raw" />
    <remap from="image_right" to="/davis_right/image_raw" />
    
    <remap from="events_left" to="/davis_left/events" />
    <remap from="events_right" to="/davis_right/events" />
    
    <remap from="camera_info_left" to="/davis_left/camera_info" />
    <remap from="camera_info_right" to="/davis_right/camera_info" />
        
    <remap from="reset_timestamps" to="/davis_left/reset_timestamps" />
  </node>
  
  <!-- display -->

	<node name="image_view_rectified_stereo" pkg="image_view" type="image_view">
    <remap from="image" to="davis_stereo/dvs_rectified"/>
  </node>

  
</launch>
